deepmd/utils/argcheck.py

```diff
@@ -356,6 +359,9 @@ def descrpt_se_atten_args():
     doc_attn_mask = "Whether to do mask on the diagonal in the attention matrix"
 
     return [
+        Argument(
+            "compressible", bool, optional=True, default=False, doc=doc_compressible
```

Comment:
Shall we change the default to `True`? and write down in the document that setting it to `False` will fall back to the old behavior of `se_atten`